queries/lib.go

```diff
@@ -0,0 +1,39 @@
+package queries
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func parseQueryFile(path string) string {
+	basePath := formatBasePath()
+	// Ensure file is a valid .sql file
```

Comment:
I wouldn't say "valid", since we aren't checking the contents or syntax of the file itself